index.js

```diff
@@ -139,6 +140,7 @@ module.exports = {
         'react/no-string-refs': 2,
         'react/no-unescaped-entities': 2,
         'react/no-unknown-property': 2,
+        'react/no-unsafe': 2,
```

Comment:
enabling new rules in an exported config is a semver-major change; please set this to 0 for now.